src/components/AddressLookup/index.tsx

```diff
@@ -133,7 +138,12 @@ const AddressLookup = forwardRef(
         })
         .then((data: { Items: AddressDataType[] }) => {
           clearOptions();
-          onAddressSelected(data.Items[0]);
+          const selectedAddress: AddressDataType = data.Items[0];
+          const selectedAddressWithoutCommas =
+            removeCommasFromObjectStringValues<AddressDataType>(
```

Comment:
[OPTIONAL] You don't need to be explicit here with the type `<AddressDataType>` as it will come from `selectedAddress`